ionic2/app/pages/wiki-index/wiki-index.page.scss

```diff
@@ -0,0 +1,290 @@
+.wiki-index {
+  background-color: #f4f4f7;
+  .pokeimg-fixed {
+    width: 100px;
+    height: 100px;
+  }
+  .card-center {
+    display: flex;
+    justify-content: center;
+  }
+}
+
+
+
+/* ==========================================================================
+    GRIDLEX
+    Just a Flexbox Grid System
```

Comment:
Wow :D Is this the only way of making a grid system? Seems like a lot of code for some boxes ;)